db/migrate/20180116133813_add_price_to_dish.rb

```diff
@@ -0,0 +1,5 @@
+class AddPriceToDish < ActiveRecord::Migration[5.1]
+  def change
+    add_column :dishes, :price, :float
```

Comment:
Why 2 migrations? Removing and adding price?

Reply:
Because we needed to change the price from integer to float

Reply:
We probably could have solved with one migration but we felt safer to first remove price as a integer and then add it back as a float